src/main/java/seedu/address/logic/commands/RemoveClassCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Classes;
+
+/**
+ * Deletes a class identified using it's displayed index from the class book.
+ */
+public class RemoveClassCommand extends Command {
+    public static final String COMMAND_WORD = "rm";
```

Comment:
Great to change remove to rm, helps our goal of making the CLI responsive for fast typers